### ISSUE TITLE
lipo error with Carthage 0.35.0

### ISSUE THREAD

**Issue by markd2** [muxinc/mux-stats-sdk-avplayer#54]
Hi! 

Since upgrading to Carthage 0.35.0, when building (iOS) with against MUXSDKStats for the iOS Simulator, we're getting a lipo error:

```
A shell task (/usr/bin/xcrun lipo -remove arm64 -output /var/folders/qt/v9_nzmxs0sg0jrhxt3bm_0rr0000gn/T/carthage-lipo-oQRCWc/MUXSDKStats.framework.dSYM/Contents/Resources/DWARF/MUXSDKStats /var/folders/qt/v9_nzmxs0sg0jrhxt3bm_0rr0000gn/T/carthage-lipo-oQRCWc/MUXSDKStats.framework.dSYM/Contents/Resources/DWARF/MUXSDKStats) failed with exit code 1:
fatal error: /Applications/Xcode-11-4-1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/lipo: -remove's specified would result in an empty fat file
```

That particular file `/var/folders/qt/v9_nzmxs0sg0jrhxt3bm_0rr0000gn/T/carthage-lipo-oQRCWc/MUXSDKStats.framework.dSYM/Contents/Resources/DWARF/MUXSDKStats` only has the arm64 side, which is what's getting removed (since the destination is for the simulator)

```
/var/folders/qt/v9_nzmxs0sg0jrhxt3bm_0rr0000gn/T/carthage-lipo-oQRCWc/MUXSDKStats.framework.dSYM/Conte\
nts/Resources/DWARF/MUXSDKStats (for architecture arm64): Mach-O 64-bit dSYM companion file arm64
```

Xcode 11.4.1, Catalina 10.15.5 (19F101). Carthage 0.35.0

And here's a simple project that reproduces the problem (includes the pulled binaries).  Or can grab from a repo from here:  https://github.com/markd2/Carthage-muxstats   (you'll need to run `carthage update`)

[Carthage-muxstats.zip](https://github.com/muxinc/mux-stats-sdk-avplayer/files/4847047/Carthage-muxstats.zip)

Thanks!

**Comment by dylanjha** [comment]
@markd2 thanks! This actually came up and I was taking a look at it the other day. Thanks for the repro steps, I'll take a look at that later today.

In the meantime, can you update your Cartfile to point to this branch: `dj/update-for-carthage-0-35` and let me know if that solves it?

**Comment by markd2** [comment]
Thanks for the rapid turnaround!  I'm a carthage n00b (just digging in to it now).   

Trying the branch, and getting a `Framework not found: MUXSDKStats`

Made this my Cartfile:

```
github "muxinc/mux-stats-sdk-avplayer" "dj/update-for-carthage-0-35"
github "muxinc/stats-sdk-objc" ~> 2.2.0
```

and did a `carthage update`

```
% carthage update
*** Fetching stats-sdk-objc
*** Fetching mux-stats-sdk-avplayer
*** Checking out stats-sdk-objc at "v2.2.2"
*** Checking out mux-stats-sdk-avplayer at "7323e9e2f8bade8f13683a12b0e40d69785847b0"
*** xcodebuild output can be found in /var/folders/qt/v9_nzmxs0sg0jrhxt3bm_0rr0000gn/T/carthage-xcodeb\
uild.fz9GoS.log
*** Downloading stats-sdk-objc.framework binary at "v2.2.2"
*** Skipped building mux-stats-sdk-avplayer due to the error:
Dependency "mux-stats-sdk-avplayer" has no shared framework schemes

If you believe this to be an error, please file an issue with the maintainers at https://github.com/mu\
xinc/mux-stats-sdk-avplayer/issues/new
```

**Comment by dylanjha** [comment]
@markd2 welp, let me take a look, I might have to create a beta release and attach the framework to it. Will check out your example project and report back, thanks!

**Comment by dylanjha** [comment]
@markd2 thanks for confirming `v1.3.7-beta.0` works. Today or tomorrow I'll get a release together for `v1.3.7`

**Comment by dylanjha** [comment]
This as been fixed in [v1.3.7](https://github.com/muxinc/mux-stats-sdk-avplayer/releases/tag/v1.3.7)
